### PR TITLE
Improves the freedom hardsuit

### PR DESCRIPTION
**Pull request by Zostroll**
Due to https://github.com/yogstation13/yogstation/pull/788 the freedom hardsuit is ingame. The tendril only drops the suit but no helmet so I made the helmet internal, gave it a internal jetpack and added flash resist to make it useful.
:cl: Zostroll
tweak: Adds a internal helmet to the freedom hardsuit, gives it a internal jetpack and makes the helmet flash resistant.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
This is a spacesuit, you might wanna make this a hardsuit so it has all the properties of them, like the nodrop helmet and all that

edit: yeah that's the problem, it needs to be defined as a hardsuit, not a space suit

**Comment by Cruix** [comment]
Yeah, it needs to be a hardsuit to have an inbuilt helmet. Also the underscore I added is wrong.

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 1]
```diff
@@ -261,12 +261,14 @@ Contains:
...
 	unacidable = 1
+	flash_protect = 1
 
-/obj/item/clothing/suit/space/freedom
+/obj/item/clothing/suit/space/hardsuit/freedom
 	name = "eagle suit"
 	desc = "An advanced, light suit, fabricated from a mixture of synthetic feathers and space-resistant material. A gun holster appears to be intergrated into the suit and the wings appear to be stuck in 'freedom' mode."
 	icon_state = "freedom"
 	item_state = "freedom"
+	helmettype = /obj/item/clothing/head/helmet/space/freedom
```
Comment (ShadowDeath6) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: /obj/item/clothing/head/helmet/space/hardsuit/freedom

Comment (Zostroll) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: I am terrible.

**Comment by Zostroll** [comment]
I gave it a built in jetpack cause why not.

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 2]
```diff
@@ -274,6 +274,10 @@ Contains:
 	strip_delay = 130
 	max_heat_protection_temperature = FIRE_IMMUNITY_HELM_MAX_TEMP_PROTECT
 	unacidable = 1
+	
+/obj/item/clothing/suit/space/hardsuit/freedom/New()
+	jetpack = new /obj/item/weapon/tank/jetpack/suit(src)
+..()
```
Comment (Ephemeralis) [inline code/modules/clothing/spacesuits/miscellaneous.dm:280]: incase you didn't see this before, your indentation is wrong here

**Pull request merged**